### ISSUE TITLE
Define the contracts Bootstrapping Release Process & Batch Migration

### ISSUE THREAD

**Issue by ct-ctrl** [ethersphere/storage-incentives#83]
**Define the plan, process and order for initialisation of the contracts and oracle setting price upon release etc.** 

**note:** batch migration should be done before the price is set to the minimum on the Oracle. During the migration make sure to not set roles or deploy other contracts or set the price. 

**question:** Do we now force set initial price to the minimum so there is no zero case? If so will this negatively affect the previous note on the batch migration process? 

**question:** the copyBatch does not fire expire(expireLimited?) or update validchunkcount - does that mean that it is only to be intended to be used during the initialisation of the contract?
validChunkCount += 1 << _depth; should it be added to copybatch?

**Comment by significance** [comment]
i agree, it would be good to document the bootstrapping process for completeness
